.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         # See https://cloud.google.com/docs/authentication/production
         command: |
           export GOOGLE_APPLICATION_CREDENTIALS="/tmp/gcp.json"
-          echo "$GCLOUD_SERVICE_KEY" > "$GOOGLE_APPLICATION_CREDENTIALS"
+          echo "$GCLOUD_SERVICE_KEY_INTEGRATION_TEST" > "$GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
`$GCLOUD_SERVICE_KEY` is by default used for pushing images to the container registry. Before it was set to `pensieve-integration-test`. I changed all CircleCI variables so that the images get, again, pushed to `experiments-analysis`. `$GCLOUD_SERVICE_KEY_INTEGRATION_TEST` is required to run integration tests in `pensieve-integration-test`